app/screens/orders/index.js

```diff
@@ -1,13 +1,139 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+// import React from 'react';
+import * as React from 'react';
+import { Text, View, ScrollView, StyleSheet }  from 'react-native';
+import styles from './styles';
 
-const Orders = () => {
+
+export default Orders = () => {
 
   return(
-  <View>
-    <Text>Orders</Text>
-  </View>
+    <ScrollView>
+            <View style={{flex:1}}>
```

Comment:
Since you already have a styles file, you can just add this there and name it as something.